managed/src/main/java/com/yugabyte/yw/cloud/AWSInitializer.java

```diff
@@ -516,8 +518,7 @@ private void storeInstanceTypeInfoToDB() {
           LOG.error(msg);
           throw new UnsupportedOperationException(msg);
         } else {
-          // TODO: hardcode me not?
-          volumeCount = 2;
+          volumeCount = YB_VOLUME_INFO_VOLUME_COUNT;
```

Comment:
It looks like you forgot to read the value from the config file.

Reply:
FYI check `TaskGarbageCollector` for good example of how to inject config and use it. Other places are using deprecated code and that will be fixed soon.

Reply:
Oops! My bad! Updated the patch. 

@sb-yb  I am not sure on how are we going to migrate DB for older instance types, need to check. Let's connect whenever your time permits to bring this to closure. Thanks!